packages/color-kit-block/src/Palette.spec.ct.tsx

```diff
@@ -0,0 +1,44 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { mount } from '@cypress/react';
+import { FrontifyColorPaletteDummy } from '@frontify/app-bridge';
+
+import { Palette } from './Palette';
+
+const ColorKitPaletteSelector = '[data-test-id="palette"]';
+const ColorKitColorSelector = '[data-test-id="color"]';
+
+const dummyPalette = FrontifyColorPaletteDummy.with(666, 'Palette Dummy');
+
+describe('Palette', () => {
+    context('with colors', () => {
+        beforeEach(() => {
+            mount(<Palette palette={dummyPalette} />);
+
+            cy.get(ColorKitPaletteSelector).should('exist').as('palette');
+            cy.get(ColorKitPaletteSelector).find(ColorKitColorSelector).should('exist').as('colors');
+        });
+
+        it('should render', () => {
+            cy.get('@colors').should('have.length', 3);
```

Comment:
not sure why this is 3, since it should check if the component is rendered

Reply:
Ah, I feel you. Let me try to explain. In the context of tests, it says _with colors_, and then this specific test renders the component (with colors). And for question why precisely three. It is because of FrontifyColorPaletteDummy, as it holds three colors. I hope this clears it for you. But yeah, I'm unsure how to write it in code to be more transparent. Suggestions are welcome.